Profiles/cio-Reaction.xml

```diff
@@ -295,58 +306,21 @@
       <min value="1" />
       <max value="1" />
     </element>
-    <element id="AllergyIntolerance.reaction.extension:exposureDateTime">
-      <path value="AllergyIntolerance.reaction.extension" />
-      <sliceName value="exposureDateTime" />
-      <type>
-        <code value="Extension" />
-        <profile value="http://hl7.org/fhir/StructureDefinition/openEHR-exposureDate" />
-      </type>
-    </element>
-    <element id="AllergyIntolerance.reaction.extension:exposureDateTime.value[x]">
-      <path value="AllergyIntolerance.reaction.extension.value[x]" />
-      <short value="ExposureDateTime" />
-      <definition value="Moment of exposure to the substance. This may be a vague date.&#xD;&#xA;&#xD;&#xA;If a user would like to provide more information about the chosen moment of exposure, they may use the comment field to do so." />
-      <comment value="Ideally, one would use the [openEHR-exposureDescription extension](http://hl7.org/fhir/StructureDefinition/openEHR-exposureDescription) to convey more information regarding the chosen moment of exposure. However, since there is only one Comment concept within the CiO data set, `.reaction.note.text` should be used instead.&#xD;&#xA;&#xD;&#xA;Note that even though this extension states it's related to the 'first exposure', it has been deemed suitable to represent the ExposureDateTime concept based on [this discussion on openEHR Discourse](https://discourse.openehr.org/t/meaning-of-initial-exposure-in-adverse-reaction-event/4679)." />
-      <mapping>
-        <identity value="cio-dataset-200-beta2-20231214" />
-        <map value="cio-dataelement-20-817" />
-        <comment value="ExposureDateTime" />
-      </mapping>
-    </element>
-    <element id="AllergyIntolerance.reaction.extension:latencyReaction">
+    <element id="AllergyIntolerance.reaction.extension:nominalValue">
       <path value="AllergyIntolerance.reaction.extension" />
-      <sliceName value="latencyReaction" />
+      <sliceName value="nominalValue" />
       <type>
         <code value="Extension" />
-        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-Reaction.LatencyReaction" />
-      </type>
-    </element>
-    <element id="AllergyIntolerance.reaction.extension:latencyReaction.value[x]">
-      <path value="AllergyIntolerance.reaction.extension.value[x]" />
-      <mapping>
-        <identity value="cio-dataset-200-beta2-20231214" />
-        <map value="cio-dataelement-20-884" />
-        <comment value="LatencyReaction" />
-      </mapping>
-    </element>
-    <element id="AllergyIntolerance.reaction.extension:durationReaction">
-      <path value="AllergyIntolerance.reaction.extension" />
-      <sliceName value="durationReaction" />
-      <type>
-        <code value="Extension" />
-        <profile value="http://hl7.org/fhir/StructureDefinition/allergyintolerance-duration" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-Reaction.NominalValue" />
```

Comment:
This extension should be renamed, as NominalValue is too generic and does not really reflect the content of the extension.